muda/deformers/background.py

```diff
@@ -29,13 +29,55 @@ def sample_clip(filename, n_samples, sr, mono=True):
     sr : int > 0
         The target sampling rate
 
+    Returns
+    -------
+    start : int
+        The sample index from `filename` at which the audio fragment starts
+    stop : int
+        The sample index from `filename` at which the audio fragment stops (e.g. y = audio[start:stop])
+    '''
+
+    with psf.SoundFile(str(filename), mode='r') as soundf:
+
+        n_target = int(np.ceil(n_samples * soundf.samplerate / sr))
```

Comment:
`sr` should be explicitly cast here as `float(sr)` to ensure floating-point division in legacy python